roles/KubevirtNodeLabeller/templates/kubevirt-node-labeller-ds.yaml.j2

```diff
@@ -15,6 +15,9 @@ spec:
       labels:
         app: kubevirt-node-labeller
     spec:
+      affinity: {{ cr_info.spec.affinity | default("", true) | to_json }}
```

Comment:
Please don't forget to update default values, same as here https://github.com/kubevirt/kubevirt-ssp-operator/blob/master/roles/KubevirtTemplateValidator/templates/service.yaml.j2#L61

Reply:
Done